app/src/main/java/org/oppia/app/options/OptionsActivity.kt

```diff
@@ -3,19 +3,27 @@ package org.oppia.app.options
 import android.content.Context
 import android.content.Intent
 import android.os.Bundle
+import android.widget.TextView
 import org.oppia.app.R
 import org.oppia.app.activity.InjectableAppCompatActivity
 import org.oppia.app.drawer.KEY_NAVIGATION_PROFILE_ID
 import javax.inject.Inject
 
+private const val SELECTED_OPTIONS_TITLE_KEY = "SELECTED_OPTIONS_TITLE_KEY"
+
 /** The activity for setting user preferences. */
 class OptionsActivity :
   InjectableAppCompatActivity(),
   RouteToAppLanguageListListener,
   RouteToAudioLanguageListListener,
-  RouteToStoryTextSizeListener {
+  RouteToStoryTextSizeListener,
+  LoadStoryTextSizeListener,
+  LoadAppLanguageListListener,
+  LoadAudioLanguageListListener {
   @Inject
   lateinit var optionActivityPresenter: OptionsActivityPresenter
+  // used to initially load the suitable fragment in the case of multipane.
+  var isFirstRun = true
```

Comment:
This should not be used directly in other files. Instead pass it down to the viewmodel via fragment/presenter.

Reply:
Done